cluster-monitoring/templates/prometheusrule.yaml

```diff
@@ -210,7 +210,7 @@ spec:
   - name: resource-usage.rules
     rules:
     - alert: ContainerExcessiveCPU
-      expr: sum(node_namespace_pod_container:container_cpu_usage_seconds_total:sum_rate) by (container, pod, namespace) / sum(kube_pod_container_resource_requests_cpu_cores) by (container, pod, namespace) > 2.0
+      expr: sum(node_namespace_pod_container:container_cpu_usage_seconds_total:sum_rate) by (container, pod, namespace) / sum(kube_pod_container_resource_requests{resource=\"cpu\", unit=\"core\"}) by (container, pod, namespace) > 2.0
```

Comment:
are you sure you need to escape the quotes here? Other `expr` in this file don't have them escaped.

Reply:
Ah no idd. They only needed escaping in the dashboard.